tests/unit/lms/events/test_event.py

```diff
@@ -121,3 +123,72 @@ def pyramid_request(self, pyramid_request):
         )
 
         return pyramid_request
+
+
+class TestAuditTrailEvent:
+    def test_model_change_from_instance(self, db_session):
+        org = factories.Organization(name="OLD_NAME", enabled=True)
+        db_session.flush()
+        org.name = "NEW_NAME"
+        org.enabled = False
+
+        assert AuditTrailEvent.ModelChange.from_instance(
+            org, action=sentinel.action, source=sentinel.source, userid=sentinel.userid
+        ) == AuditTrailEvent.ModelChange(
+            model="Organization",
+            id=Any.int(),
+            source=sentinel.source,
+            action=sentinel.action,
+            userid=sentinel.userid,
+            changes={"name": ("OLD_NAME", "NEW_NAME"), "enabled": (True, False)},
+        )
+
+    def test__get_data(self, pyramid_request):
+        change = AuditTrailEvent.ModelChange(
+            id=sentinel.id,
+            model=sentinel.model,
+            source=sentinel.source,
+            action=sentinel.action,
+            userid=sentinel.userid,
+            changes=sentinel.changes,
+        )
+
+        event = AuditTrailEvent(request=pyramid_request, change=change)
+
+        assert event.data == {
+            "id": sentinel.id,
+            "model": sentinel.model,
+            "source": sentinel.source,
+            "action": sentinel.action,
+            "userid": sentinel.userid,
+            "changes": sentinel.changes,
+        }
```

Comment:
How about something like this?

```suggestion
        kwargs = {
            "id": sentinel.id,
            "model": sentinel.model,
            "source": sentinel.source,
            "action": sentinel.action,
            "userid": sentinel.userid,
            "changes": sentinel.changes,
        }

        event = AuditTrailEvent(
            request=pyramid_request, change=AuditTrailEvent.ModelChange(**kwargs)
        )

        assert event.data == kwargs
```